src/hotspot/share/opto/loopnode.cpp

```diff
@@ -310,7 +310,7 @@ IdealLoopTree* PhaseIdealLoop::insert_outer_loop(IdealLoopTree* loop, LoopNode*
 // clone the outer loop as well as the inner, unrolling needs to only
 // clone the inner loop etc. No optimizations need to change the outer
 // strip mined loop as it is only a skeleton.
-IdealLoopTree* PhaseIdealLoop::create_outer_strip_mined_loop(BoolNode *test, Node *cmp, Node *init_control,
+IdealLoopTree* PhaseIdealLoop::create_outer_strip_mined_loop(Node *init_control,
```

Comment:
Generally, for the touched code, you can also fix the `*` placement to be at the type.
```suggestion
IdealLoopTree* PhaseIdealLoop::create_outer_strip_mined_loop(Node* init_control,
```

Reply:
Good point. I tried to fix them as I go but missed this one.

Reply:
With the `x` -> `head` change, there are some more opportunities :-)